sale_project_user_profile/__manifest__.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    'name': 'Sale Project User Profile',
+    'version': '10.0.1.0.0',
+    'author': 'Camptocamp SA,Odoo Community Association (OCA)',
+    'license': 'AGPL-3',
+    'category': 'Project',
+    'website': 'http://www.camptocamp.com',
```

Comment:
Follow new OCA guidelines regarding website in manifest: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#modules